linera-service/tests/end_to_end_tests.rs

```diff
@@ -304,6 +343,90 @@ async fn test_wallet_lock() -> Result<()> {
     Ok(())
 }
 
+#[test_case(LocalNetConfig::new_test(Database::Service, Network::Grpc) ; "service_grpc")]
+#[cfg_attr(feature = "scylladb", test_case(LocalNetConfig::new_eth(Database::ScyllaDb, Network::Grpc) ; "scylladb_grpc"))]
+#[cfg_attr(feature = "dynamodb", test_case(LocalNetConfig::new_eth(Database::DynamoDb, Network::Grpc) ; "aws_grpc"))]
+#[test_log::test(tokio::test)]
+async fn test_wasm_end_to_end_ethereum_tracker(config: impl LineraNetConfig) -> Result<()> {
+    use alloy::primitives::U256;
+    use ethereum_tracker::{EthereumTrackerAbi, InstantiationArgument};
+    use linera_ethereum::test_utils::{get_anvil, SimpleTokenContractFunction};
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+
+    let anvil_test = get_anvil().await?;
+    let address0 = anvil_test.get_address(0);
+    let address1 = anvil_test.get_address(1);
+    let ethereum_endpoint = anvil_test.endpoint.clone();
+
+    let simple_token = SimpleTokenContractFunction::new(anvil_test).await?;
+    let contract_address = simple_token.contract_address.clone();
+    let argument = InstantiationArgument {
+        ethereum_endpoint,
+        contract_address,
+    };
+
+    let (_net, client) = config.instantiate().await?;
+    let chain = client.load_wallet()?.default_chain().unwrap();
+
+    // Change the ownership so that the blocks inserted are not
+    // fast blocks. Fast blocks are not allowed for the oracles.
+    let pub_key1 = {
+        let wallet = client.load_wallet()?;
+        let user_chain = wallet.get(chain).unwrap();
+        user_chain.key_pair.as_ref().unwrap().public()
+    };
+    let pub_key2 = PublicKey::test_key(2);
```

Comment:
Why not just a single owner? (It just has to be regular, not super, then it won't make fast blocks.)

Reply:
Ok.